fw/thread.go

```diff
@@ -382,11 +387,11 @@ func (t *Thread) processIncomingData(pendingPacket *ndn.PendingPacket) {
 
 	// Add to Content Store
 	if t.pitCS.IsCsAdmitting() {
-		t.pitCS.InsertDataCS(data)
+		t.pitCS.InsertData(data)
 	}
 
 	// Check for matching PIT entries
-	pitEntries := t.pitCS.FindPITFromData(data, pitToken)
+	pitEntries := t.pitCS.FindInterestPrefixMatchByData(data, pitToken)
```

Comment:
`FindPITFromData` was not originally planned to exist as a function for the `PitCsTable` interface but here, in `processIncomingData`, we only have access to data and not interests, so we need to also allow users to look up PIT entries using data, in addition to looking using an interest.